packages/gatsby-theme-api-docs/src/components/resource-method.js

```diff
@@ -21,24 +20,22 @@ const ResourceMethod = ({ apiKey, resource, method, title }) => {
   }
 
   return (
-    <FullWidthContainer>
```

Comment:
FullWidthContainer moved directly into the "Method" component so that the "Method" can directly serve as the "pass your own data" component without having to provide yet another wrapper